src/site/layouts/health_care_region_page.drupal.liquid

```diff
@@ -138,73 +140,108 @@ Example data:
       <article class="usa-content">
         <h1>{{ title }}</h1>
         {% assign image = fieldMedia.entity.image %}
-        <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="{{ image.derivative.width }}" height="{{ image.derivative.height }}">
-        <div class="usa-grid usa-grid-full">
-          <div class="usa-width-one-third">
-            Make an appointment
+        <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="100%">
```

Comment:
This image isn't coming through for me locally, the `src` attribute ends up being empty

Reply:
Yeah, the image doesn't exist on stg, and it gets lost on dev whenever there's a merge on the drupal code.